src/sentry/interfaces/sdk.py

```diff
@@ -61,6 +61,23 @@ def get_api_context(self):
                 # when this is correct we can make it available
                 # 'version': newest_version,
                 'isNewer': is_newer,
-                'url': settings.SDK_URLS.get(newest_name),
+                'url': self.get_with_prefix(settings.SDK_URLS, newest_name),
             },
         }
+
+    @staticmethod
```

Comment:
I'm ok with this, but we shouldn't make this a staticmethod. There's no value in tying it to the class here. Just write a function at the module level.